managed-services/transactional-email/how-to/bimi.mdx

```diff
@@ -0,0 +1,60 @@
+---
+meta:
+  title: How to set up BIMI for Transactional Emails
+  description: Learn how to set up a Brand Indicators for Message Identification (BIMI) record to enhance email authentication and brand visibility.
+content:
+  h1: How to set up BIMI for Transactional Emails
+  paragraph: Learn how to set up a Brand Indicators for Message Identification (BIMI) record to enhance email authentication and brand visibility.
+tags: dmarc spf dkim bimi brand txt record dns
+dates:
+  validation: 2024-07-10
+  posted: 2024-07-10
+categories:
+  - managed-services
+---
+
+Scaleway supports SPF, DKIM and DMARC with Transactional Email, which makes it possible for your to set up BIMI, or Brand Indicators for Message Identification. BIMI is an email specification that allows you to use your brand's logo within email clients to help demonstrate the authenticity of your emails. BIMI helps reduce the risk of phishing and spoofing, for example, as brand indicators are used as a tool to identify if fraud is occurring.
+
+## Before you start
+
+To complete the actions presented below, you must have:
+
+- A Scaleway account logged into the [console](https://console.scaleway.com/)
+- [Configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).
+- A domain name that passed [SPF](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), [DKIM](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), and [DMARC](/managed-services/transactional-email/how-to/add-dmarc-record) authentication checks
+- A logo of your brand saved in SVG format
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status of the Organization in which the actions will be performed, or be an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+
+## How does BIMI work?
+
+Before your brand's logo can be displayed, your email must first pass authentication checks. Checks are usually performed through [SPF (Sender Policy Framework)](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), [DKIM (DomainKeys Identified Mail)](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), and [DMARC (Domain-based Message Authentication, Reporting, and Conformance)](/managed-services/transactional-email/how-to/add-dmarc-record) checks, which are supported by Scaleway. Some email clients also require a Verified Mark Certificate (VMC) to ensure the logo's authenticity.
+
+BIMI is a DNS TXT record that you set up to indicate what brand's logo. The record contains the URL to the brand's logo in SVG (Scalable Vector Graphics) format.
```

Comment:
I don't understand what you mean with the first sentence, I think something's missing.